.github/actions/system-info/system-info.py

```diff
@@ -0,0 +1,32 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+import platform
+import sys
+
+print(f"{sys.platform=}", file=sys.stderr)
+
+if sys.platform == "linux":
+    platform_info = platform.freedesktop_os_release()
+    release = platform_info["VERSION_ID"]
```

Comment:
According to the docs `VERSION_ID` is optional. Only `NAME`, `ID`, and `PRETTY_NAME` are always present.
I guess this shouldn't be an issue if the CI runs only on Ubuntu, but maybe we should add a comment like:
```suggestion
    release = platform_info["VERSION_ID"]  # not available in all platforms
```
Or maybe something like:
```suggestion
    release = platform_info.get("VERSION_ID", "unknown")
```

BTW, on my config:
```
>>> print(json.dumps(platform.freedesktop_os_release(), indent=4))
{
    "NAME": "Debian GNU/Linux",
    "ID": "debian",
    "PRETTY_NAME": "Debian GNU/Linux trixie/sid",
    "VERSION_CODENAME": "trixie",
    "HOME_URL": "https://www.debian.org/",
    "SUPPORT_URL": "https://www.debian.org/support",
    "BUG_REPORT_URL": "https://bugs.debian.org/"
}
```

Reply:
done